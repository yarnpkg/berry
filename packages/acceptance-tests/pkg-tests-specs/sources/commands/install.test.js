describe(`Commands`, () => {
  describe(`add`, () => {
    test(
      `it should print the logs to the standard output when using --inline-builds`,
      makeTemporaryEnv({
        dependencies: {
          [`no-deps-scripted`]: `1.0.0`,
        },
      }, async ({path, run, source}) => {
        const {stdout} = await run(`install`, `--inline-builds`);

        await expect(stdout).toMatchSnapshot();
      }),
    );
  });
});
