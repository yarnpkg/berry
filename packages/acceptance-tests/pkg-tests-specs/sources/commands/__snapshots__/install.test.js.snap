// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Commands add it should print the logs to the standard output when using --inline-builds 1`] = `
"➤ YN0000: ┌ Resolution step
➤ YN0000: └ Completed
➤ YN0000: ┌ Fetch step
➤ YN0013: │ no-deps-scripted@npm:1.0.0 can't be found in the cache and will be fetched from the remote registry
➤ YN0000: └ Completed
➤ YN0000: ┌ Link step
➤ YN0007: │ no-deps-scripted@npm:1.0.0 must be built because it never did before or the last one failed
➤ YN0000: │ no-deps-scripted@npm:1.0.0 STDOUT preinstall out
➤ YN0000: │ no-deps-scripted@npm:1.0.0 STDERR preinstall err
➤ YN0000: │ no-deps-scripted@npm:1.0.0 STDOUT install out
➤ YN0000: │ no-deps-scripted@npm:1.0.0 STDERR install err
➤ YN0000: │ no-deps-scripted@npm:1.0.0 STDOUT postinstall out
➤ YN0000: │ no-deps-scripted@npm:1.0.0 STDERR postinstall err
➤ YN0000: └ Completed
➤ YN0000: Done
"
`;
